sdk/storage/storage-blob-changefeed/src/BlobChangeFeedClient.ts

```diff
@@ -0,0 +1,115 @@
+import { BlobServiceClient } from "@azure/storage-blob";
+import { PagedAsyncIterableIterator, PageSettings } from "@azure/core-paging";
+import { BlobChangeFeedEvent } from "./models/BlobChangeFeedEvent";
+import { ChangeFeedFactory } from "./ChangeFeedFactory";
+import { ChangeFeed } from "./ChangeFeed";
+import { CHANGE_FEED_DEFAULT_PAGE_SIZE } from "./utils/constants";
+
+export interface ChangeFeedGetChangesOptions {
+  start?: Date;
+  end?: Date;
+}
+
+export class BlobChangeFeedEventPage {
+  public events: BlobChangeFeedEvent[];
+  public continuationToken: string;
+
+  constructor() {
+    this.events = [];
+    this.continuationToken = "";
+  }
+}
+
+export class BlobChangeFeedClient {
+  /**
+   * blobServiceClient provided by @azure/storage-blob package.
+   *
+   * @private
+   * @type {BlobServiceClient}
+   * @memberof DataLakeServiceClient
```

Comment:
minor: please check the doc comments all through the code, e.g. here is inaccurate.

Reply:
Fixed